cmd/controller/ctrl.go

```diff
@@ -106,7 +116,7 @@ func createDR(dr apis.Disk, c *Controller) {
 			 * update the resource version after updation is succssful here, the update call
 			 * from that node will fail.
 			 */
-			glog.Info("disk status updated by other node, changing the ownership to this node : ", err)
+			glog.Info("Disk status updated by other node, changing the ownership to this node : ", err)
```

Comment:
Should this be an Info or an Err?

Reply:
This log is a info.

Reply:
Same comment as above

Reply:
We are using info as this is not exactly a error case we just put an info for debug using log.

Reply:
changed to warning